library/common/config_template.cc

```diff
@@ -1,5 +1,7 @@
 /**
  * Templated default configuration
+ *
+ * Note: Parts of this configuration are required for the direct API to function.
```

Comment:
This seems like a given since the library only provides a direct API now. I'd either remove this or make it clearer as to which parts of the config are required for a specific purpose

Reply:
That's fair; I just wanted to call it out somewhere for folks who might plan to pass their own configuration.

I could comment on the clusters themselves down in the YAML.

Reply:
updated